pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features.adoc

```diff
@@ -0,0 +1,162 @@
+= Implementing Your Custom Policy Features in Rust
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+The following pages provide Rust code examples that complete various policy tasks. Implement the Rust code for your policy in the `src/lib.rs` file.
+
+The code examples assume you have some prior knowledge of the Rust Programming Language. For more information about programming in Rust, see:
+
+* https://doc.rust-lang.org/book/[The Rust Programming Language Documentation ^]
+* https://rust-lang-nursery.github.io/rust-cookbook/intro.html[Rust Cookbook ^]
+
+== Before You Begin
```

Comment:
This implies that the doc is a tutorial, but there are no steps to follow. I read all the following info as optional things they can do, which means that this isn't a tutorial.